src/it/scala/io/iohk/ethereum/txExecTest/ForksTest.scala

```diff
@@ -24,6 +24,8 @@ class ForksTest extends FlatSpec with Matchers {
     override val daoForkBlockNumber: BigInt = 10000
     override val daoForkBlockHash: ByteString = ByteString("unused")
     override val daoForkBlockTotalDifficulty: BigInt = 1000000
+    override val difficultyBombPauseBlockNumber: BigInt = Long.MaxValue
+    override val difficultyBombContinueBlockNumber: BigInt = Long.MaxValue
```

Comment:
Not here. Although this may actually be good idea to export some post ecip1010 blocks from parity and try to execute them as integration tests. But for that I'd probably need to sync from scratch with `--pruning archive` is that right @adamsmo ?